IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypeCatenaryStay/DocPropertySet.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypeCatenaryStay" UniqueId="b15eff21-bd3f-df08-8b2a-e9722e73b14d" ApplicableType="IfcMember/(.STAY_CABLE.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
> ApplicableType="IfcMember/(.STAY_CABLE.)"

I dont think this encoding is correct? @fhulin did this come out of IFC doc or was manually changed?
@SergejMuhic @jmirtsch bug?

Reply:
Done.c0f5835a24d8f1939a80ada93c3b1cb04772002d